docs/release-notes.md

```diff
@@ -1,3 +1,11 @@
+## 0.20.0
+
+April 28, 2022
+
+### Removed
+* Drop Python 3.6 support [#1357](https://github.com/encode/starlette/pull/1357).
```

Comment:
Shouldn't the checklist be added here?

Reply:
What items do you think should be included?

Reply:
We don't have written rules for the changelog on starlette, but tests, docs and type hints modifications were never included.

Reply:
Yeah I meant the doc changes, but you're right, they were never part of release changes.